src/ds_caselaw_utils/data/schema/courts.schema.json

```diff
@@ -88,7 +88,17 @@
             }
           },
           "required": ["code", "name", "link", "listable", "selectable"],
-          "additionalProperties": false
+          "additionalProperties": false,
+          "allOf": [
```

Comment:
not entirely sure how this schema is utilised, but can we test this somehow?

Reply:
At the moment it's run using the `check-jsonschema` action in pre-commit, which checks that the schema itself is valid and then checks that the `court_names.yaml` validates against it.

We could test it, but we'd need to build a test harness and craft a test fixture with a known problem, and then verify that validating it returns the expected error. Doable, just not convinced of the utility.